project/PATCH-RELEASES.md

```diff
@@ -0,0 +1,53 @@
+# Docker patch (bugfix) release process
+
+Patch releases (the 'Z' in vX.Y.Z) are intended to fix major issues in a
+release. Docker open source projects follow these procedures when creating a
```

Comment:
@thaJeztah In that case, I would not mention other docker projects in this document.